### PR TITLE
Fixes plastic flaps

### LINKED ISSUE #5031: (WEB REPORT BY: Anonymous REMOTE: ) Reinforcing the floor under airtight flaps makes them not airtight
>Revision 9ca1158533c2a8c0bb9bce5d4550ca0c867e4126

>General description of the issue
When rods are used to reinforce the floor under airtight flaps, they lose their airtightness. You can easily observe this at the cargo shuttle conveyors.

### PR DESCRIPTION
**Pull request by clusterfack**
Fixes #5031

-Regular plastic flaps were not originally meant to be airtight, they can now be made airtight with a crowbar and anchored/unanchored with a wrench.
-They now use canpass instead of EDITING THE TURF AIR BELOW IT to set airtight qualities.
-They now have examine text and a different name/desc to show if they are anchored/airtight or not
-Wrench anchors/unanchors as it does with most equipment

### CODE REVIEW

**Comment by 9600bauds** [comment]
What do you mean "regular plastic flaps"
There is more than one kind?

**Comment by clusterfack** [comment]
Originally there were plastic flaps, and airtight flaps.Airtight flaps were used in mining and they were the only ones that were airtight.

Kurfurst made plastic flaps constructible, but for some reason made it so when you construct them they are airtight, unlike plastic flaps that start spawned on the map.

File: code/game/supplyshuttle.dm [hunk 1]
```diff
@@ -81,7 +88,7 @@ var/list/mechtoys = list(
 		var/mob/living/M = mover
 		if(!M.lying && !istype(M, /mob/living/carbon/monkey) && !istype(M, /mob/living/carbon/slime) && !istype(M, /mob/living/simple_animal/mouse))  //If your not laying down, or a small creature, no pass.
 			return 0
-	return ..()
+	return !airtight
```
Comment (ComicIronic) [inline code/game/supplyshuttle.dm]: return !(air_group && airtight)

You'll block crates with this.

Comment (clusterfack) [inline code/game/supplyshuttle.dm]: Alright

**Comment by clusterfack** [comment]
Done

**Comment by d3athrow** [comment]
CHANGELOG

**Comment by clusterfack** [comment]
Changelog for my recent changes is done

**Pull request merged**